src/Processors/Formats/Impl/ParquetBlockOutputFormat.cpp

```diff
@@ -70,6 +70,68 @@ ParquetBlockOutputFormat::ParquetBlockOutputFormat(WriteBuffer & out_, const Blo
 }
 
 void ParquetBlockOutputFormat::consume(Chunk chunk)
+{
+    /// Do something like SquashingTransform to produce big enough row groups.
+    /// Because the real SquashingTransform is only used for INSERT, not for SELECT ... INTO OUTFILE.
+    /// The latter doesn't even have a pipeline where a transform could be inserted, so it's more
```

Comment:
But why SELECT INTO OUTFILE does not have a pipeline?
Let's ask @KochetovNicolai 
... I have an impression that we are using pipelines everywhere.

Reply:
This would need to be a different pipeline, on the client side, right? Currently ClientBase just receives a block from network and puts it into IOutputFormat, simple.